src/content/docs/release-notes/agent-release-notes/python-release-notes/python-agent-90800.mdx

```diff
@@ -0,0 +1,66 @@
+---
+subject: Python agent
+releaseDate: '2024-03-27'
+version: 9.8.0
+downloadLink: 'https://pypi.python.org/pypi/newrelic'
+features: ['Add support for asgiref 3.8.0 and above', 'AI monitoring configuration', 'AI monitoring public API methods', 'AI monitoring', 'Add support for OpenAI', 'Add support for Langchain', 'Add support for AWS Bedrock']
+bugs: []
+security: []
+---
+
+## Notes
+
+This release of the Python agent adds support for the latest versions of [asgiref](https://pypi.org/project/asgiref/) and support for AI monitoring when using the following libraries: [OpenAI](https://pypi.org/project/openai/), [AWS Bedrock](https://pypi.org/project/boto3/), and [Langchain](https://pypi.org/project/langchain/).
+
+Install the agent using `easy_install/pip/distribute` via the [Python Package Index](https://pypi.python.org/pypi/newrelic) or download it directly from the [New Relic download site](https://download.newrelic.com/python_agent/release).
+
+
+## New features
+
+* **Add support for [asgiref](https://pypi.org/project/asgiref/) 3.8.0 and above**
+
+    *  Asgiref released a new version that resulted in missing transaction context. This has been fixed.
+
+* **AI monitoring configuration**
+
+    * New configuration options are available specific to [AI monitoring](https://docs.newrelic.com/docs/apm/agents/python-agent/configuration/python-agent-configuration/#ai-monitoring). These settings include:
+        * ai_monitoring.enabled 
+        * ai_monitoring.streaming.enabled
+        * ai_monitoring.content.enabled
+
+* **AI monitoring public API methods**
+
+    * Two new AI monitoring related public API methods have been added:
+        * [record_llm_feedback_event](https://docs.newrelic.com/docs/apm/agents/python-agent/python-agent-api/recordllmfeedbackevent-python-agent-api/)
+        * [set_llm_token_count_callback](https://docs.newrelic.com/docs/apm/agents/python-agent/python-agent-api/setllmtokencountcallback-python-agent-api/)
+
+* **AI monitoring**
```

Comment:
super nitpick but wondering if we should move this main AI monitoring section up above the API and config sections so that the context is there first before mentioning the other features added for AIM?